.travis.yml

```diff
@@ -52,7 +53,7 @@ jobs:
       language: python
       python:
         - "3.8"
-      if: (branch = master OR type = pull_request) AND fork = false
+      if: branch = master OR tag =~ /^release.*$/
```

Comment:
it should run on all branches not only on master if you remove PR

Reply:
Done.